11/bullseye/Dockerfile

```
@@ -33,7 +33,7 @@ RUN jlink \
          --compress=2 \
          --output /javaruntime
 
-FROM debian:bullseye-20210816
+FROM debian:bullseye
```

I prefer the date stamp on the Docker image so that we track how close we are to the current date.  Is there a compelling reason why this needs to remove the date code?
```suggestion
FROM debian:bullseye-20210816
```

---

dependabot doesn't work with it and we don't have enough maintainers.
We have a higher chance of staying up to date if we don't fix it to this tag.

but if you feel strongly I can change back

---

can do a follow up if required